aura/doc/aura.8

```diff
@@ -145,7 +145,7 @@ suppressed for a more silent install. Note that when this option isn't used,
 \fImakepkg\fR output is actually collected and printed if any errors occur.
 .RE
 .P
-\fB\-\-json <package(s)>
+\fB\-\-json\fP \fI<package(s)>\fP
```

Comment:
There are other flags that have `<package(s>` etc as arguments, but they don't seem to be highlighted. Should they be?

Reply:
As I wrote in my top post, I'm humbly requesting your permission to reformat all those flags at once using **groff_man**(7) requests like `.BI`. Would be too much work for lazy me to first highlight them using current (inline) code style, and then rewrite to requests.

Reply:
For now, I could just revert the `\fI` part.